site2/docs/schema-get-started.md

```diff
@@ -0,0 +1,55 @@
+---
+id: schema-get-started
+title: Get started
+sidebar_label: Get started
+---
+
+Pulsar does not parse data, and it takes bytes as inputs and sends bytes as outputs. While data has meaning beyond bytes, you need to parse data and might encounter parse exceptions which mainly occur in the following situations:
```

Comment:
"Pulsar does not parse data" => when schema is enabled, Pulsar does parse data though.